internal/server/resolve_objection.go

```diff
@@ -0,0 +1,99 @@
+package server
+
+import (
+	"fmt"
+	"github.com/ministryofjustice/opg-go-common/template"
+	"github.com/ministryofjustice/opg-sirius-lpa-frontend/internal/sirius"
+	"net/http"
+)
+
+type ResolveObjectionClient interface {
+	GetObjection(sirius.Context, string) (sirius.Objection, error)
+	ResolveObjection(sirius.Context, string, string, sirius.ResolutionRequest) error
+}
+
+type resolveObjectionData struct {
+	XSRFToken   string
+	Success     bool
+	Error       sirius.ValidationError
+	Errors      map[string]sirius.ValidationError
```

Comment:
Since the name is very close to "Error", might be worth naming it something more distinctive, e.g.
```suggestion
	ErrorsByCase map[string]sirius.ValidationError
```